### ISSUE TITLE
Wake up does not work

### ISSUE THREAD

**Issue by BoerndDreams** [sharandac/My-TTGO-Watch#385]
**Build tool used:**

- [X] Platformio (Visual Studio Code plugin)
- [ ] Platformio CLI
- [ ] Other

**used Hardware:**

- [X] TWatch 2020 or TWatch 2021
- [ ] M5Paper
- [ ] M5Core2
- [ ] WT32-SC01

**Description of problem:**
Unfortunately, the wake up by double click, tilt does not work.

**Additional information and things you've tried:**
Settings checked, reinstalled, rebooted, no changes in source code

**Comment by TimelessNL** [comment]
+1 on this issue.
Recently acquired a TWatch 2020 V3 and tilt to wake does not work on my watch either.

**Comment by kukata86** [comment]
Yes wake up with tilt work on v1 but not in v3, any solution until now.? Please advice
